src/components/JsonSchemaValidator/dbtUnified.js

```diff
@@ -316,8 +316,9 @@ export const dbtSnowplowUnifiedConfigSchema = {
           type: 'object',
           title: "Identifier",
           properties: {
-            schema: { type: 'string' }, // TODO: add regex here to make valid context/unstruct or atomic?
-            field: { type: 'string' } // TODO: add regex here to make valid SQL name?
+            schema: { type: 'string', description: 'the schema name of your events table, atomic in most use cases, alternatively for sdes/contexts this should instead be the name of the field itself' }, // TODO: add regex here to make valid context/unstruct or atomic?
+            field: { type: 'string', description: 'the name of the field to use as session identifier, alternatively, in case of sdes/contexts it is the name of the element that refers to the field to be extracted' } // TODO: add regex here to make valid SQL name?
+          },
```

Comment:
Start desc with capital please. Also it will need the prefix field added as you noted in the other response.